### ISSUE TITLE
Sandbox: Invalid protocol constants

### ISSUE THREAD

**Issue by beatalipska** [trilitech/jstz#465]
# Context
During my walkthrough the getting started guide, after using `jstz sandbox start` I have encountered a following error:
```
⠂ [█████████████████░░░░░░░░░░░░░░░░░░░░░░░]       7/16   
ERROR  Command TEZOS_CLIENT_UNSAFE_DISABLE_DISCLAIMER="Y" "octez-client" "--base-dir" 
"/tmp/octez_clientK7efGN" "--endpoint" "http://127.0.0.1:18730" "-block" "genesis" "activate" "protocol" 
"ProtoALphaALphaALphaALphaALphaALphaALphaALphaDdp3zK" "with" "fitness" "1" "and" "key" "activator" "and" "parameters" "/usr/share/jstz/sandbox-params.json" failed:
Error: Invalid protocol constants: The DAL committee must be a subset of the Tenderbake committee.
```
NPM version: 9.8.1
Docker version 24.0.6, build ed223bc820

**Comment by johnyob** [comment]
Hey 👋 

I've managed to reproduce this locally -- it appears that the sandbox parameters for this weeks build of octez changed. 
I'll make a PR now to patch this. 

As a quick workaround, I suggest using weeklynet for deploying your smart function, which you can do by adding `--network weeklynet` flag to the `deploy` and `run` commands.

**Comment by johnyob** [comment]
Will be fixed by #467 for docker images and #446 for those building from sources

**Comment by johnyob** [comment]
Images have been built and pushed. Updating the images using the installer:
```sh
source <(curl --proto '=https' --tlsv1.2 -sSf https://raw.githubusercontent.com/trilitech/jstz/main/scripts/install-jstz-cli.sh)
```
will now fix this issue :)

**Comment by alanmarkoTrilitech** [comment]
fixed by #467 and #446
